packages/pn-commons/src/utils/notification.utility.ts

```diff
@@ -115,19 +115,21 @@ export const NotificationAllowedStatus = [
 
 /**
  * Returns the mapping between current notification timeline status and its label and descriptive message.
- * @param  {TimelineCategory} status
+ * @param  {INotificationDetailTimeline} step
+ * @param {Array<NotificationDetailRecipient>} recipients
  * @returns object
  */
 export function getNotificationTimelineStatusInfos(
   step: INotificationDetailTimeline,
-  ricipients: Array<NotificationDetailRecipient>
+  recipients: Array<NotificationDetailRecipient>
 ): {
   label: string;
   description: string;
   linkText?: string;
   recipient?: string;
 } | null {
   const recipient = !_.isNil(step.details.recIndex) ? ricipients[step.details.recIndex] : undefined;
```

Comment:
rinominare ricipients in recipients

Reply:
corretto, mi è sfuggito durante il merge